### PR TITLE
chore(docs): add amplify cli instructions to liveness docs

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Added cli instructions from [rek docs](https://docs.aws.amazon.com/rekognition/latest/dg/face-liveness-prerequisites.html#face-liveness-prerequisites-amplify) to the ui docs site

Screenshot 2023-05-05 at 5 30 36 PM

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by zchenwei — APPROVED** [review]
🚢

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 1]
```diff
@@ -6,7 +6,54 @@ There are three methods to setup the Amazon Cognito resources needed for Amplify
...
+```
+
+#### Add the auth category
+
+```bash
+$ amplify add auth
+ Do you want to use the default authentication and security configuration? Manual configuration
+ 
+ Select the authentication/authorization services that you want to use: User Sign-Up, Sign-In, connected with AWS IAM controls (Enables per-user Stora
+ge features for images or other content, Analytics, and more)
```
Comment (wlee221) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:34]:
```suggestion
 Select the authentication/authorization services that you want to use: User Sign-Up, Sign-In, connected with AWS IAM controls (Enables per-user Storage features for images or other content, Analytics, and more)
```

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx [hunk 2]
```diff
@@ -6,7 +6,54 @@ There are three methods to setup the Amazon Cognito resources needed for Amplify
...
+? Distribution Directory Path: build
+? Build Command:  npm run-script build
+? Start Command: npm run-script start
+```
+
+#### Add the auth category
+
+```bash
+$ amplify add auth
+ Do you want to use the default authentication and security configuration? Manual configuration
```
Comment (wlee221) [inline docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:31]: Do we need to use manual configuration?

**Review by wlee221 — APPROVED** [review]

**Review by wlee221 — APPROVED** [review]

File: docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx (hunk 2 above)
Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/quick-start-pull.react.mdx:31]: yep it's required from the rekognition docs as well just so they can set up unauthenticated/guest access

**Review by tjleing — APPROVED** [review]

**Pull request merged**